### PR TITLE
[Artwork] Add RelatedWorks test

### PR DESCRIPTION
**Pull request by damassi**
Adds a small test to RelatedArtworks. 

(Will do a test sweep to add more coverage in a cleanup pass across app.)

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b20f48549d4e7665bee567358791ffcba6dfa8e2

https://deploy-preview-1656--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/RelatedWorksArtworkGrid.tsx [hunk 1]
```diff
@@ -23,16 +23,19 @@ const logger = createLogger("RelatedWorksArtworkGrid.tsx")
 
 const MAX_TAB_ITEMS = 3
 
-interface Props {
+interface RelatedWorksArtworkGridProps {
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/RelatedWorksArtworkGrid.tsx]: any reason for this? sometimes the namespacing can be a bit...tedious. I've been using `Props` and `State` in the file, for the local container/component props and state.

Also semi-related to a PR discussion with @oxaudo on naming/organization of the various elements in a file (query renderer, fragment container, react component, props/state, occasional wrappers to add mediator and other things) - https://github.com/artsy/reaction/pull/1646#discussion_r239610662

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/RelatedWorksArtworkGrid.test.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  }
+
+  let wrapper: ReactWrapper
+
+  describe("rendering", () => {
+    beforeAll(async () => {
+      wrapper = await getWrapper()
+    })
+
+    it("renders the artist name", () => {
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/__tests__/RelatedWorksArtworkGrid.test.tsx]: 🍝

File: src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/RelatedWorksArtworkGrid.tsx (hunk 1 above)
Comment (damassi) [reply src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/RelatedWorksArtworkGrid.tsx]: Mostly just for consistency with other code -- technically speaking the only time I naming needs to be this precise is when the code is being imported from another application where `d.ts` files are automatically generated.

**Pull request merged**